server/tests/tasks/test_github_repo_import.py

```
@@ -0,0 +1,154 @@
+
+import routes.github.github_repo_metadata as mod
+
+_USER = "user-1"
+_INSTALL = 4242
+
+
+class _MaxRetriesExceeded(Exception):
+    """Stand-in for Celery's ``self.MaxRetriesExceededError`` exception class."""
+
+
+def _task_self():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_task_self`

(ANN202)